zipline/transforms/batch_transform.py

```diff
@@ -34,12 +34,7 @@
 
 from zipline.utils.data import MutableIndexRollingPanel
 from zipline.protocol import Event
-from zipline.finance.trading import TradingEnvironment
-
-# HACK the BatchTransform module stores a trading environment to be used by
-# the transforms
-# TODO remove this hack, if not this whole module
-_batch_transform_env = TradingEnvironment()
```

Comment:
We chatted about possibly using ```get_algo_instance().trading_environment``` instead of duplicating the code from ```TradingEnvironment``` here.  And if that's too difficult for now, we should add a new comment about why we're duplicating it.